### PR TITLE
[SCAL-80045]-Solved Stuttering issue

### PR DESCRIPTION
**Pull request by aayushgupta6**

### CODE REVIEW

**Review by ayonghosh — APPROVED** [review]

File: docs/src/pages/index.tsx [hunk 1]
```diff
@@ -148,8 +148,9 @@ const IndexPage = ({ location }) => {
     };
 
     return (
-        <>
+        <>  
             <main ref={ref as React.RefObject<HTMLDivElement>}>
+
```
Comment (pankajp-ts) [inline docs/src/pages/index.tsx:153]: remove this new line `(line 153)`

File: docs/src/pages/index.tsx [hunk 2]
```diff
@@ -170,13 +172,16 @@ const IndexPage = ({ location }) => {
                         options={results}
                         optionSelected={optionSelected}
                     />
+                    </Suspense>
                     <div className="introWrapper">
+                        <Suspense fallback={<div> </div>}>
```
Comment (pankajp-ts) [inline docs/src/pages/index.tsx:177]: do we need space between tag `<div> </div>` if not please remove it.

**Review by pankajp-ts — CHANGES_REQUESTED** [review]

File: docs/src/pages/index.tsx (hunk 2 above)
Comment (aayushgupta6) [reply docs/src/pages/index.tsx]: No, I'll remove it

File: docs/src/pages/index.tsx (hunk 1 above)
Comment (aayushgupta6) [reply docs/src/pages/index.tsx]: ok

File: docs/src/pages/index.tsx [hunk 3]
```diff
@@ -148,8 +148,9 @@ const IndexPage = ({ location }) => {
     };
 
     return (
-        <>
+        <>
```
[the remark this answers is not in the document]
Comment (aayushgupta6) [reply docs/src/pages/index.tsx]: ok

File: docs/src/pages/index.tsx [hunk 4]
```diff
@@ -1,13 +1,13 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState, useEffect,Suspense,lazy } from 'react';
```
[the remark this answers is not in the document]
Comment (aayushgupta6) [reply docs/src/pages/index.tsx]: ok

**Review by pankajp-ts — APPROVED** [review]
LGTM 👍

**Pull request merged**